projects/private-app-getting-started-template/package.json

```diff
@@ -1,11 +1,7 @@
 {
```

Comment:
I wonder if it would it be beneficial to remove this file entirely?  Running `npm install` at the wrong directory level seemed to be a pretty common support issue, and the post install was mitigating this in at least some of the cases. Are there benefits to keeping this file around without the `scripts` block?

Reply:
I could not agree more, @joe-yeager! I took a baby step to remove the fangs from this issue, but I am also happy if we go all the way and remove it!